src/Services/DatabaseTools/AbstractDatabaseTool.php

```diff
@@ -61,7 +61,10 @@ abstract class AbstractDatabaseTool
      */
     protected $om;
 
-    protected Connection $connection;
+    /**
+     * @var Connection
```

Comment:
It might be to also add `MongoDB\Client` to the type declaration here so it's not assumed this is only the DBAL's connection in the future.